src/dmd/astbase.d

```diff
@@ -6056,9 +6067,9 @@ struct ASTBase
             return null;
         }
 
-        ExpInitializer isExpInitializer()
+        final ExpInitializer isExpInitializer()
         {
-            return null;
+            return kind == InitKind.exp ? cast(ExpInitializer)cast(void*)this : null;
```

Comment:
Since those are `C++` classes, aren't the `void*` cast useless ?

Reply:
You're right.

Reply:
But it doesn't cause any problems, and if they are ever converted to D classes, adding the cast might be overlooked, and would be a hidden speed bump. I suppose I could argue it either way.